File: src/server/Mimirorg.TypeLibrary/Models/Client/LibraryTypeLibCm.cs
namespace Mimirorg.TypeLibrary.Models.Client
{
    public class LibraryTypeLibCm
    {
        public IEnumerable<NodeLibCm> Nodes { get; set; }
        public IEnumerable<InterfaceLibCm> Interfaces { get; set; }
        public IEnumerable<TransportLibCm> Transports { get; set; }
        public IEnumerable<SubProjectLibCm> SubProjects { get; set; }
    }
}

File: src/client/src/models/typeLibrary/client/libraryTypeLibCm.ts
import { NodeLibCm } from "./nodeLibCm";
import { InterfaceLibCm } from "./interfaceLibCm";
import { TransportLibCm } from "./transportLibCm";

export interface LibraryTypeLibCm {
  nodes: NodeLibCm[];
  interfaces: InterfaceLibCm[];
  transports: TransportLibCm[];
  kind: string;
}
